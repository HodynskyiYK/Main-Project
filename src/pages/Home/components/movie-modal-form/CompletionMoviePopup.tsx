import React, { FC } from 'react'
import styles from './MovieModalForm.module.scss'
import { COMPLETION_ICON_66_66 } from '../../../../assets/svg/CompletionIcon'
import classnames from 'classnames'

interface ICompletionMoviePopup {
    completionText: string
}

export const CompletionMoviePopup: FC<ICompletionMoviePopup> = ({completionText}) => {
    return (
        <div className={styles.completionMovie}>
            <div className={styles.completionIcon}>{
                COMPLETION_ICON_66_66
            }</div>
            <p className={classnames('h1', styles.completionTitle)}>congratulations !</p>
            <p className={styles.completionText}>{completionText}</p>
        </div>
    )
}
