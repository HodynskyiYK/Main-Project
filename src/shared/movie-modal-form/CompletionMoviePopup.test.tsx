import React from 'react'
import { render } from '@testing-library/react'
import { CompletionMoviePopup } from './CompletionMoviePopup'

describe('CompletionMoviePopup tests', () => {

    test('To match snapshot', () => {
        const {asFragment} = render(<CompletionMoviePopup completionText={'Some text'}/>)
        expect(asFragment()).toMatchSnapshot()
    })
})