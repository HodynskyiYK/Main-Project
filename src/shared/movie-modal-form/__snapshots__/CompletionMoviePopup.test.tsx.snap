// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompletionMoviePopup tests To match snapshot 1`] = `
<DocumentFragment>
  <div
    class="completionMovie"
  >
    <div
      class="completionIcon"
    >
      <svg
        fill="none"
        height="66"
        viewBox="0 0 66 66"
        width="66"
        xmlns="http://www.w3.org/2000/svg"
      >
        <circle
          cx="33"
          cy="33"
          fill="#F65261"
          r="32.5"
          stroke="#F65261"
        />
        <path
          d="M14 35.8347L24.1175 46L49 21"
          stroke="white"
          stroke-linecap="round"
          stroke-width="5"
        />
      </svg>
    </div>
    <p
      class="h1 completionTitle"
    >
      congratulations !
    </p>
    <p
      class="completionText"
    >
      Some text
    </p>
  </div>
</DocumentFragment>
`;
